src/semgrep_agent/semgrep_app.py

```diff
@@ -156,10 +156,10 @@ def fetch_rules_text(self) -> str:
             return response.text
 
     def download_rules(self) -> Tuple[Path, List[str], List[str]]:
-        """Save the rules configured on semgrep app to a temporary file"""
-        # hey, it's just a tiny YAML file in CI, we'll survive without cleanup
-        rules_file = tempfile.NamedTemporaryFile(suffix=".yml", delete=False)  # nosem
-        rules_path = Path(rules_file.name)  # nosem
+        """Save the rules configured on semgrep app to a file in .semgrep_logs"""
+        """so that it persists for debugging """
+        rules_file = SEMGREP_RULES_FILE
+        rules_path = Path(rules_file)  # nosem
```

Comment:
I assume the nosem is not needed since we're not using `tempfile`'s `.name` anymore.

```suggestion
        rules_path = Path(rules_file)
```